testsuite/tests/service_mesh/auth/wasm/test_authorities.py

```diff
@@ -0,0 +1,79 @@
+"""Tests wasm authorities filtering"""
+from urllib.parse import urlsplit
+from typing import List
+import pytest
+from testsuite.capabilities import Capability
+from testsuite.gateways.wasm import ServiceMeshHttpClient
+
+
+pytestmark = pytest.mark.required_capabilities(Capability.SERVICE_MESH_WASM)
+
+
+@pytest.fixture(scope="module")
+def client_ingress(api_client, extension) -> ServiceMeshHttpClient:
+    """Returns api client using default ingress_url as ingress to service mesh."""
+    client: ServiceMeshHttpClient = api_client()
+    client.root_url = extension.ingress_url  # redundant, as its default, but included for better readability
+    return client
+
+
+@pytest.fixture(scope="module")
+def client_alias(api_client, extension) -> ServiceMeshHttpClient:
+    """Returns api client using first alias url as ingress to service mesh."""
+    client: ServiceMeshHttpClient = api_client()
+    client.root_url = extension.ingress_alias_url
+    return client
+
+
+def test_alias(extension, client_ingress, client_alias):
+    """This test sets extension authorities to only accept alias host."""
+    extension.replace_authorities([urlsplit(extension.ingress_alias_url).netloc])
```

Comment:
This seems dangerous as the tests cannot be executed in parallel (Should not cause problems for now as we run each module together). How hard would it be to create an extension per test?

Reply:
Would be just bit more complicated. 
By creating `service` per test there will be `extension` per test. But than every test should also have its own `client_alias` as alias url is unique per extension. I will rewrite it that way than.

Reply:
I did rewrite the test the way I described above. It has drawback of longer test run because of additional service/extension creations and rollouts.